### PR TITLE
chore(infra): unify config package

### PR DESCRIPTION
**Pull request by barsukov**
# Summary

Many lilttle leftovers but very important ones, fix main brunch typecheck.

# Changes Made

- Delete eslintconfig package
- FIX OAUTH leftovers types
- Fix docs
- Add typecheck CI
- Change k8s output
- Delete tsconfig package
- create juno-config pacakge
- Changesets

# Related Issues

- Issue 1: #274 

# Testing Instructions

1. `npm i`
2. `npm run TASK`

# Checklist

- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have made corresponding changes to the documentation (if applicable).
- [x] My changes generate no new warnings or errors.

### CODE REVIEW

File: packages/config/README.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+- Typescript
+- ESlint
+
+## Typescript
+
+This package provides a base TypeScript configuration to be used across multiple packages in a monorepo setup. It ensures consistent TypeScript settings and simplifies the TypeScript setup process for each package.
+
+## Usage
+
+You can use this configuration as a starting point in any package or app within your monorepo using TurboRepo.
```
Comment (edda) [inline packages/config/README.md:14]: Does this sentence mean that other people could use this setup in their own monorepos? Or did you mean "package or app with **this** monorepo?

File: packages/config/README.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+```
+
+## ESLint
+
+This package provides also a temporary ESLint configuration specifically for TypeScript to be used across multiple packages in a monorepo setup.
+
+## Usage
+
+You can use this configuration as a starting point in any package or app within your monorepo using TurboRepo.
```
Comment (edda) [inline packages/config/README.md:40]: Same question as above

File: packages/config/README.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+    "outDir": "./build",
+    "rootDir": "./src"
+  },
+  "include": ["src"]
+}
+```
+
+## ESLint
+
+This package provides also a temporary ESLint configuration specifically for TypeScript to be used across multiple packages in a monorepo setup.
```
Comment (edda) [inline packages/config/README.md:36]: What does "temporary" mean in this context?

**Review by edda — CHANGES_REQUESTED** [review]

File: packages/config/README.md (hunk 1 above)
Comment (barsukov) [reply packages/config/README.md:14]: package or app within our monorepo

File: packages/config/README.md (hunk 3 above)
Comment (barsukov) [reply packages/config/README.md:36]: Cause we need two configs one for typescript one for old javascript yet. But I will delete word temporary

**Review by ArtieReus — APPROVED** [review]
lgtm

**Review by andypf — APPROVED** [review]
LGTM

**Review by edda — APPROVED** [review]

**Pull request merged**